{"abstract":[{"type":"text","text":"An error connecting to Stripe’s API."}],"metadata":{"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"apiConnection","kind":"identifier"}],"modules":[{"name":"Stripe"}],"symbolKind":"case","title":"STPSetupIntentLastSetupErrorType.apiConnection","externalID":"c:@M@StripePayments@E@STPSetupIntentLastSetupErrorType@STPSetupIntentLastSetupErrorTypeAPIConnection","roleHeading":"Case"},"hierarchy":{"paths":[["doc:\/\/com.stripe.stripe-ios\/documentation\/Stripe","doc:\/\/com.stripe.stripe-ios\/documentation\/Stripe\/STPSetupIntentLastSetupErrorType"]]},"identifier":{"url":"doc:\/\/com.stripe.stripe-ios\/documentation\/Stripe\/STPSetupIntentLastSetupErrorType\/apiConnection","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"attribute","text":"@objc"},{"text":"(STPSetupIntentLastSetupErrorTypeAPIConnection) ","kind":"text"},{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"apiConnection"}],"platforms":["iOS"]}],"kind":"declarations"}],"kind":"symbol","schemaVersion":{"minor":3,"patch":0,"major":0},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/stripe\/stpsetupintentlastsetuperrortype\/apiconnection"]}],"references":{"doc://com.stripe.stripe-ios/documentation/Stripe":{"title":"Stripe","kind":"symbol","identifier":"doc:\/\/com.stripe.stripe-ios\/documentation\/Stripe","role":"collection","type":"topic","abstract":[],"url":"\/documentation\/stripe"},"doc://com.stripe.stripe-ios/documentation/Stripe/STPSetupIntentLastSetupErrorType":{"type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"STPSetupIntentLastSetupErrorType","kind":"identifier"}],"navigatorTitle":[{"text":"STPSetupIntentLastSetupErrorType","kind":"identifier"}],"abstract":[{"text":"The type of the error represented by ","type":"text"},{"type":"codeVoice","code":"STPSetupIntentLastSetupError"},{"type":"text","text":"."},{"type":"text","text":" "},{"text":"Some STPSetupIntentLastError properties are only populated for certain error types.","type":"text"}],"kind":"symbol","url":"\/documentation\/stripe\/stpsetupintentlastsetuperrortype","role":"symbol","identifier":"doc:\/\/com.stripe.stripe-ios\/documentation\/Stripe\/STPSetupIntentLastSetupErrorType","title":"STPSetupIntentLastSetupErrorType"},"doc://com.stripe.stripe-ios/documentation/Stripe/STPSetupIntentLastSetupErrorType/apiConnection":{"kind":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"apiConnection"}],"url":"\/documentation\/stripe\/stpsetupintentlastsetuperrortype\/apiconnection","role":"symbol","type":"topic","identifier":"doc:\/\/com.stripe.stripe-ios\/documentation\/Stripe\/STPSetupIntentLastSetupErrorType\/apiConnection","abstract":[{"type":"text","text":"An error connecting to Stripe’s API."}],"title":"STPSetupIntentLastSetupErrorType.apiConnection"}}}